src/tutorial/scheduling/integer-cycling.rst

```diff
@@ -362,29 +362,7 @@ Recurrence Sections
            # Repeat every third cycle.
            P3 = baz
 
-.. digraph:: example
-   :align: center
-
-   subgraph cluster_1 {
-       label = 1
-       style = dashed
-       "foo.1" [label="foo\n1"]
-       "bar.1" [label="bar\n1"]
-       "baz.1" [label="baz\n1"]
-   }
-
-   subgraph cluster_2 {
-       label = 2
-       style = dashed
-       "foo.2" [label="foo\n2"]
-   }
-
-   subgraph cluster_3 {
-       label = 3
-       style = dashed
-       "foo.3" [label="foo\n3"]
-       "bar.3" [label="bar\n3"]
-   }
+.. image:: ../../img/reccurrance-sections.svg
```

Comment:
Could you check this fits on both the slides and the HTML page, you can play about with `:width:` (or is it `:size:` I can't remember) on the `..image::` directive otherwise.

Could you stick an `:align: center` on this.

Reply:
Yes. and Yes. Will do.

Reply:
@oliver-sanders Can you re-check this.